dotcom-rendering/src/client/userFeatures/user-features.ts

```diff
@@ -51,19 +52,25 @@ const refresh = async (): Promise<void> => {
 	}
 };
 
-const requestNewData = () => {
-	return getAuthStatus()
-		.then((authStatus) =>
-			authStatus.kind === 'SignedInWithCookies' ||
-			authStatus.kind === 'SignedInWithOkta'
-				? authStatus
-				: Promise.reject('The user is not signed in'),
-		)
-		.then((signedInAuthStatus) => {
-			return syncDataFromMembersDataApi(signedInAuthStatus).then(
-				persistResponse,
-			);
-		});
+const shouldUseUserBenefitsApi = (): boolean => {
+	return true;
+	// const abTestAPI = useAB()?.api;
+	// return !!abTestAPI?.isUserInVariant('UserBenefitsApi', 'variant');
```

Comment:
did you mean to leave that commented out, I think this will bring in 100% as it is

Reply:
Yes I'm just doing this to test on CODE, I should probably just opt in to it via the url though!